style.css

```diff
@@ -756,3 +868,273 @@ textarea {
     padding-left: 150px;
   }
 }
+
+@media screen and (max-width: 768px) {
+  .navbar-des {
+    display: none;
+  }
+
+  .bar-btn {
+    border: none;
+    background-color: white;
+    padding: 10px;
+  }
+
+  .headline-container {
+    overflow-y: auto;
+  }
+
+  /* Mobile-menu */
+  .mobile-menu {
+    position: fixed;
+    width: 100%;
+    height: 100%;
+    z-index: 5;
+    top: 0;
+    left: 0;
+    background-color: #f7f7f9;
+  }
+
+  .mobile-menu-container {
+    position: fixed;
+    display: flex;
+    gap: 30px;
+    background-color: white;
+    width: 100%;
+    top: 0;
+    left: 0;
+    height: 90vh;
+    border: none;
+    border-bottom-left-radius: 30px;
+    border-bottom-right-radius: 30px;
+  }
+
+  .close-btn {
+    position: absolute;
+    margin: 10px;
+    top: 30px;
+    right: 40px;
+    z-index: 9;
+    display: flex;
+    justify-content: flex-end;
+    color: var(--close-icon-color);
+  }
+
+  .mobile-menu-list {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    margin-top: 70px;
+    margin-left: 10px;
+    padding: 20px;
+    gap: 30px;
+    color: var(--mobile-menu-items-color);
+  }
+
+  .mobile-menu-list-items {
+    font-style: normal;
+    font-weight: 600;
+    font-size: 32px;
+    line-height: 44px;
+    list-style: none;
+  }
+
+  .mobile-menu-list-items a {
+    text-decoration: none;
+    color: #3a4a42;
+  }
+
+  /* mobile detail */
+  .detail {
+    height: calc(100vh - 50px);
+    max-width: 920px;
+    left: 32px;
+    right: 32px;
+    top: 50%;
+    transform: translate(0, -50%);
+  }
+
+  .detail-container {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    padding: 5px;
+    gap: 12px;
+    width: 100%;
+    height: 100%;
+  }
```

Comment:
Kindly update your popup window on your desktop version to match the figma design better🙏

Your design:
<img width="433" alt="image" src="https://github.com/axcell-0/microverse-portfolio/assets/37116322/323ccaff-2189-463d-bc39-56ed88892842">

Figma design:
<img width="428" alt="image" src="https://github.com/axcell-0/microverse-portfolio/assets/37116322/8b112922-d97c-463c-9b69-059ed94aba1e">

A few things to note:
- Kindly make sure your modal has the proper margins as seen in the figma design
- The grey background behind the modal has some images and elements in your design which are not present in the figma design. Please update your styles so they don't show
- Kindly update your image styling so it spans the full length of the modal and is aligned with the buttons below it
- Kindly increase the font size of the main title of the modal

Reply:
As suggested by the previous reviewer, kindly update your mobile and desktop popup view to be similar to the Figma template given as it has a lot of differences from your current design, paying attention to details is one of the highly sought after skill of a programmer 😄 

Your desktop design  :

![image](https://github.com/axcell-0/microverse-portfolio/assets/20537712/ee132ec6-8542-4b60-a7eb-561a1faafae4)

what it should look like :
![image](https://github.com/axcell-0/microverse-portfolio/assets/20537712/82d6e653-0c2f-4b40-bed8-73da5426e2cb)

Your mobile design :
![image](https://github.com/axcell-0/microverse-portfolio/assets/20537712/b42ead1b-1e19-46c6-8fef-77b7e6162775)

What it should look like :
![image](https://github.com/axcell-0/microverse-portfolio/assets/20537712/bd20153a-2488-491d-87a6-ee93bc326065)